src/params.lean

```diff
@@ -28,16 +46,22 @@ open params
 
 variables [params.{u}]
 
+/-- To allow Lean's type checker to see that the ordering `Λr` is a well-ordering without having to
+explicitly write `Λwf` everywhere, we declare it as an instance.
+We can then deduce from the well-ordering that `Λ` is linearly ordered. -/
 instance : is_well_order Λ Λr := Λwf
 instance : linear_order Λ := linear_order_of_STO' Λr
 
+-- Since `μ` has cofinality `≥ κ`, the cardinality of `κ` must be at most that of `μ`.
```

Comment:
Do you want this to just be a regular comment? Also I don't think we should document *everything*. Docstrings are a way to redirect attention to the declarations that need it. If everything has docstrings, this doesn't work anymore.

Reply:
I disagree, this is an unfortunate consequence of the status quo. It is totally reasonable to have comments on every single definition and theorem.

Reply:
I wasn't aware of the different comment types in lean - I'll try to make my other changes consistent with these suggestions.

It is standard practice in other programming languages to provide documentation strings for all non-private/protected functions, if only to restate its purpose in English, so that's what I've tried to do here. I think we can still pull attention to declarations that need more information by mentioning them in module docstrings (like in litter.lean with the `## Main declarations` section).